abnf/odata-abnf-testcases.xml

```diff
@@ -364,6 +364,10 @@
     <Match>MonetaryAmount</Match>
   </Constraint>
 
+  <TestCase Name="Verifying the generated parser" Rule="namespace">
+    <Input>This.Should.Not.Fail</Input>
+  </TestCase>
+  

```

Comment:
Problem here are these "parser hints": they restrict what the parser accepts as namespace parts, so all of the words in the test namespace have to be in this list - none of them is right now:

https://github.com/oasis-tcs/odata-abnf/blob/6d0bff12c99986b94242966667958c18d7809b8a/abnf/odata-abnf-testcases.xml#L223-L232

Reply:
Hello Ralf, thanks again for your explanation!

I'll withdraw by PR accordingly.

I will try to disambiguate the grammar some other way.